pom.xml

```diff
@@ -236,7 +236,9 @@ THE SOFTWARE.
               <goal>execute</goal>
             </goals>
             <configuration>
-              <source>${pom.basedir}/src/main/preset-data/package.groovy</source>
+              <scripts>
+                <script>file:///${pom.basedir}/src/main/preset-data/package.groovy</script>
```

Comment:
Ouch, hope this works on Windows and in directories with spaces in their names. Constructing `file`-protocol URLs via string concatenation (or, conversely, extracting the file path using string matching) is notoriously unsafe. You need to use proper methods like `File.toUri` or (better, on UNC paths) `Path.toUri`. Oh well, guess it works well enough.

Reply:
> Ouch, hope this works on Windows and in directories with spaces in their names.

Can't speak to Windows, but it does work in directories with spaces in their names:

```
[INFO] --- gmavenplus-plugin:1.7.1:execute (preset-packager) @ jenkins-test-harness ---
[INFO] Using Groovy 2.4.8 to perform execute.
[INFO] Running Groovy script from file:////home/basil/work/third-party/jenkinsci/jenkins test harness/src/main/preset-data/package.groovy.
```

Reply:
Maybe whatever code is _using_ the URL is similarly lax, and two wrongs make a right!